### DISCUSSION TITLE
Support for mismatched resolution monitors

### DISCUSSION THREAD

**Discussion by tmdarlan** [f1multiviewer/issue-tracker#47] [Feature requests]
It appears that when a stream is dragged to a monitor with a different resolution then the main display. when using the right click menu to select the size it selects the incorrect size. when i select 1/3 of the window it should be 1/3 of the resolution of the monitor it is on. bonus points if it changes when moved from one monitor to the other.

**Comment by jeppeolesen** [comment]
Hey @tmdarlan 

If this is still relevant, please create it as an issue over here: https://github.com/f1multiviewer/issue-tracker/issues
